android/src/main/java/com/tailscale/ipn/ui/viewModel/IpnViewModel.kt

```diff
@@ -102,21 +109,24 @@ open class IpnViewModel : ViewModel() {
       }
     }
 
-    Client(viewModelScope).start(options) { start ->
-      start.onFailure { completionHandler(Result.failure(it)) }.onSuccess { loginAction() }
+    val startAction = {
+      Client(viewModelScope).start(options) { start ->
+        start.onFailure { completionHandler(Result.failure(it)) }.onSuccess { loginAction() }
```

Comment:
if this is the style, that's fine, but I personally think it's sad that onFailure and onSuccess's bodies don't line up in any way.

Reply:
Yeah, that's the auto-formatter.